3.5.3/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM rocker/rstudio:3.5.3
+
+#For some R packages you need the corresponding development library to successfully install the R package.
+RUN apt-get update -y \
+  && apt-get install -y --no-install-recommends \
+  libcurl4-openssl-dev \
+  mesa-common-dev \
+  libcgal-dev \
+  libglu1-mesa-dev \
+  libx11-dev \
+  libxml2-dev \
+  gdal-bin \
+  libgdal-dev \
+  libproj-dev \
+  libcairo2-dev \
+  libgeos-dev \
+  libgeos++-dev \
+  libudunits2-dev \
+  libpng-dev \
+  libnetcdf-dev \
+  netcdf-bin \
+  libssl-dev \
+  libssh2-1-dev \
+  libsodium-dev \
+  libv8-dev \
+  python3-dev \
+  && apt-get clean
+
+# Install Base Packages
+RUN R -e "install.packages(c('devtools','dplyr', 'knitr', 'magrittr', 'packrat', 'renv' ), repos='http://cran.rstudio.com/')"
```

Comment:
Shouldn't the 3.5.3 image be unchanged, so that nothing breaks for people already using it? The old image had a lot more R packages installed.

Reply:
I don't think that this is much of a problem, the team that we baked in all of the specific packages for isn't currently using DataLabs and everyone I am aware of are using packrat.